### PR TITLE
Add documentation for the settings UI

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

**Comment by vigejolla** [comment]
Only the [Add documentation for the settings UI](https://github.com/sailfishos/docs.sailfishos.org/pull/291/commits/f3f2d16469c89fd06da374a8c87cff3708e1c73d) commit is to be reviewed here; the earlier one is part of previous pull request.

File: Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/Settings_UI/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+The implementation must:
+
+1. Set the initialPage property (i.e. the first page in the account flow) to a page
+2. Emit accountDeletionRequested() signal and pop the page when the user wants to delete the account.
+
+The accountId property will automatically be set to the ID of the account to be displayed.
+
+### Account creation UI
+
+The UI for crating an account must be located at `/usr/share/accounts/ui/` and named `<provider>.qml` where `<provider>` matches the [name](https://sailfishos.org/develop/docs/sailfish-accounts/qml-sailfishaccounts-provider.html/#name-prop) of the [Provider](https://sailfishos.org/develop/docs/sailfish-accounts/qml-sailfishaccounts-provider.html/).
```
Comment (jovirkku) [inline Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/Settings_UI/README.md:24]: creating

File: Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/README.md [hunk 2]
```diff
@@ -29,4 +29,4 @@ Currently, no concrete roadmap for this work has yet been defined. Contributions
 
 ### Account plugins
 
-New account types can be added to the system via plugins. A plugin should describe the services it provides using [description files](Providers_and_Services/).
+New account types can be added to the system via plugins. A plugin should describe the services it provides using [description files](Providers_and_Services/). The plugin should also provide [Settings UI](Settings_UI/).
```
Comment (rainemak) [inline Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/README.md:32]: I'm tempted to say that we should have these without conditionality. Further, let's emphasize that it's Accounts Settings user interface.

My 2 cents,
"A plugin describes the services it provides using [description files](Providers_and_Services/) and provides [Accounts Settings UI integration](Settings_UI/)." Integration or components, hmm?

File: Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/Settings_UI/README.md (hunk 1 above)
Comment (vigejolla) [reply Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/Settings_UI/README.md]: fixed

**Review by rainemak — COMMENTED** [review]
Add a minor comment and it should touch https://github.com/sailfishos/docs.sailfishos.org/pull/290 as well.

**Review by rainemak — APPROVED** [review]

**Pull request merged**